FOP native support #1642 (#1793)
apache/camel-quarkus: extensions/fop/runtime/src/main/doc/limitations.adoc

While you can use any of the available output types in JVM mode, only PDF output type is supported
in native mode. PDF output type in native mode has several limitations:

> * Default sRGB color space is disabled because of https://github.com/oracle/graal/issues/2850[Graal VM issue #2850]. This limitation makes configuration property
> `disable-srgb-colorspace` ignored. You can see more about sRGB color space in
> https://xmlgraphics.apache.org/fop/2.1/configuration.html[FOP configuration].

Reviewer: ```suggestion
* Default sRGB color space is always disabled because of https://github.com/oracle/graal/issues/2850[Graal VM issue #2850] and `disable-srgb-colorspace` https://xmlgraphics.apache.org/fop/2.1/configuration.html[FOP configuration property] is ignored.
```

Author: done

* If custom fonts are used, font cache has to be disabled because of https://github.com/oracle/graal/issues/460[Graal VM issue #460].

> Please use property `<use-cache>false</use-cache>` (more information in https://xmlgraphics.apache.org/fop/2.1/configuration.html[FOP configuration].)

Reviewer: ```suggestion
Please set the https://xmlgraphics.apache.org/fop/2.1/configuration.html[FOP configuration property] `use-cache` to `false`.
```

Author: done

Author: done
